frontend/src/component/events/EventTimeline/EventTimeline.tsx

```diff
@@ -184,6 +185,7 @@ export const EventTimeline = () => {
     const startDate = sub(endDate, timeSpan.value);
     const endTime = endDate.getTime();
     const startTime = startDate.getTime();
+    const showSignalsLink = !useUiFlag('frontendHeaderRedesign');
```

Comment:
Unused and can be removed.

Reply:
Oh, you're right! Thought the linter would pick it up. But addressed in 5dc4399f9449252063afb92c970dad7cba013af7